octoprint_SpoolManager/api/SpoolManagerAPI.py

```diff
@@ -319,7 +343,12 @@ def select_spool(self):
 	def selectSpoolByQRCode(self, databaseId):
 		self._logger.info("API select spool by QR code" + str(databaseId))
 
-		spoolModel = self._selectSpool(databaseId)
+		if self._printer.is_printing():
+			# not doing this mid-print since we can't ask the user what to do
```

Comment:
mhm! I just felt like adding that since i added the message that asks what to do… before that it lways did the same thing so this was not a problem. but now i didn't want to leave it with undefined/undocumented behavior

Reply:
Currently I have no idea how many people are using the QR-Feature, so I create an issue (#163) to fix this later.